src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.hpp

```diff
@@ -141,25 +146,29 @@ class TypeLookupManager
             fastrtps::rtps::ParticipantProxyData* pdata);
 
     /**
-     * Retrieve additional type dependencies associated with a sequence of TypeIdentifiers
-     * @param id_seq
+     * Create and send a request using the builtin TypeLookup Service to retrieve all the type dependencies
+     * associated with a sequence of TypeIdentifiers.
+     * @param id_seq[in] Sequence of TypeIdentifiers for which dependencies are needed.
+     * @return The SampleIdentity of the request sended.
      */
     fastrtps::rtps::SampleIdentity get_type_dependencies(
             const xtypes::TypeIdentifierSeq& id_seq) const;
 
     /**
-     * Retrieve TypeObjects associated with a sequence of TypeIdentifiers
-     * @param id_seq
+     * Create and send a request using the builtin TypeLookup Service to retrieve TypeObjects associated with a
+     * sequence of TypeIdentifiers.
+     * @param id_seq[in] Sequence of TypeIdentifiers for which TypeObjects are to be retrieved.
+     * @return The SampleIdentity of the request sended.
      */
     fastrtps::rtps::SampleIdentity get_types(
             const xtypes::TypeIdentifierSeq& id_seq) const;
 
     /**
      * Use builtin TypeLookup service to solve the type and dependencies of a given TypeInformation.
-     * Callback is used to notify when negotiation is complete.
-     * @param typeinformation
-     * @param type_server
-     * @param callback
+     * It receives a callback that will be used to notify when the negotiation is complete.
+     * @param typeinformation[in] TypeInformation that requires solving.
+     * @param type_server[in] GuidPrefix of the WriterProxyData with the TypeInformation.
```

Comment:
`GuidPrefix corresponding to the remote participant which TypeInformation is being solved`.

Reply:
Changed in [9cf4645](https://github.com/eProsima/Fast-DDS/pull/4126/commits/9cf46451cbc0cc26f54f25c7979ada4904c22109)